molecule/default/prepare.yml

```diff
@@ -1,19 +1,29 @@
 ---
 - name: Prepare
-  hosts: all
+  hosts: all 
```

Comment:
it should be use `localhost` in hosts: ..., we didn't provision test on other environment.